syntax = "proto3";

package database;

service DatabaseService {
    rpc Read(ReadRequest) returns (ReadResponse);
    rpc Write(WriteRequest) returns (WriteResponse);
}

message ReadRequest {
    string title = 1;
}

message ReadResponse {
    int32 stockValue = 1;
}

message WriteRequest {
    string title = 1;
    int32 stockValue = 2;
}

message WriteResponse {
    int32 stockValue = 1;
}